webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -15919,6 +15919,14 @@ var userOverride = {
                             '#mck-sidebox-content'
                         );
                         var tabId = $mck_message_inner.data('mck-id');
+                        if (resp.message.metadata.actionRequest == 'askCSAT') {
```

Comment:
if we ever change something then and miss to change at any one of the place, then bugs will come,  call the same function code from both places

document.getElementById('km-csat-trigger').onclick = function (
                    e
                ) {
                    e.preventDefault();
                    KommunicateUI.triggerCSAT();
                    kommunicateCommons.modifyClassList(
                        { id: ['mck-csat-close'] },
                        'vis',
                        'n-vis'
                    );
                };

Reply:
Here, In `kommunicateCommons.modifyClassList()` we are adding `vis ` class in my case we are adding `n-vis` class.
There is one more occurence for what you pointing for have fixed that.